hadoop/README.md

```diff
@@ -0,0 +1,39 @@
+Table of contents
+* [Introduction](#Introduction)
+* [Hadoop Cluster](#Hadoop-Cluster)
+* [Hive Project](#Hive-Project)
+* [Improvements](#Improvements)
+
+
+# Introduction
+- Purpose of this project: help data processing team to process the data using Apache Hadoop  
```

Comment:
data analytics team

Reply:
I updated the Hadoop question page in case you don't have it in your scrum board.
https://www.notion.so/jarviscanada/Hadoop-Questions-984650c32c8d47ed870052b8c9e8bda7